.clang-tidy

```diff
@@ -12,13 +12,21 @@ Checks: |
   -cppcoreguidelines-pro-bounds-pointer-arithmetic
   -cppcoreguidelines-init-variables
   -cppcoreguidelines-pro-type-member-init
+  -cppcoreguidelines-avoid-do-while
+  -cppcoreguidelines-pro-type-const-cast
+  -cppcoreguidelines-pro-bounds-array-to-pointer-decay
+  -cppcoreguidelines-avoid-const-or-ref-data-members
+  -cppcoreguidelines-pro-bounds-constant-array-index
```

Comment:
I don't see this one showing up anywhere in the output 😕

Reply:
that one seems present:
```
/home/runner/work/celeritas/celeritas/src/celeritas/em/params/WentzelOKVIParams.cc:948:12: warning: do not use array subscript when the index is not an integer constant expression [cppcoreguidelines-pro-bounds-constant-array-index]
    return mott_coeffs[index];
```